README.md

```diff
@@ -26,11 +26,14 @@ CONOPS/Rules:
 **Build system**: CMake to set up build environment then whatever CMake gives you (depends on platform, make for Linux, etc.)
 
 **Dependencies**: 
-    [OpenCV 3.0](http://opencv.org/downloads.html),
+    [OpenCV 3.0](https://opencv.org/opencv-3-0.html),
```

Comment:
Perhaps better to point them to the releases page (https://opencv.org/releases.html). We should probably also say 3.x rather than 3.0 anyway, as I don't think we've had any issues with newer versions (though I don't think I've checked since 3.4 came out).